src/platform-includes/performance/opentelemetry-setup/ruby.mdx

```diff
@@ -0,0 +1,23 @@
+Initialize Sentry for tracing and set the `instrumenter` to `:otel`.
+
+```ruby {filename:config/initializers/sentry.rb}
+Sentry.init do |config|
+  config.dsn = "__DSN__"
+  config.traces_sample_rate = 1.0
+  # set the instrumenter to use OpenTelemetry instead of Sentry
+  config.instrumenter = :otel
+end
+```
+
+This will disable all Sentry instrumentation and rely on the chosen OpenTelemetry tracers for creating spans.
```

Comment:
Grammatically, this isn't quite correct. What will rely on OTEL? Your app? The SDK?
```suggestion
This disables all Sentry instrumentation and relies on the chosen OpenTelemetry tracers for creating spans.
```

Reply:
The OpenTelemetry SDK generates an spans that the Sentry SDK consumes. The Sentry SDK then takes those spans, transforms them into the Sentry format, and sends it to Sentry.

How does this sound?

```suggestion
This disables all Sentry performance instrumentation and uses the OpenTelemetry SDK to generate spans.
```

Reply:
Without getting into the nitty gritty of the grammar, how about this:
```suggestion
This disables all Sentry performance instrumentation so that the Sentry SDK relies on the chosen OpenTelemetry tracers to generate spans.
```